### DISCUSSION TITLE
dde-dock don't start in the boot

### DISCUSSION THREAD

**Discussion by Cardellino** [linuxdeepin/developer-center#3451] [General | 常规讨论]
Hello, since an update from September 26th, 2022 the dock no longer works.
Get the error message:
dde-dock -v
Warning: force enable D_DXCB_FORCE_NO_TITLEBAR now!
"dtkwidget" can not find qm files ("dtkwidget_de_DE.qm", "dtkwidget_de.qm")
translatePath after feedback: "/usr/share/dde-dock/translations/dde-dock_de"
Attribute Qt::AA_EnableHighDpiScaling must be set before QCoreApplication is created.
dde-dock 2.0

https://github.com/linuxdeepin/developer-center/issues/1552
it doesn't work....

Thanks for your support

**Comment by BLumia** [comment]
Are you using Arch Linux?

**Reply by Cardellino** [reply]
Sorry BLumia.....
My system is:
ArchLinux x86_64
5.19.12-arch1-1
Deepin 20.6
Mutter(DeepinGala)

**Reply by BLumia** [reply]
I saw users also facing this issue during a recent update. For a workaround solution, you can add https://www.archlinuxcn.org/archlinux-cn-repo-and-mirror/ archlinuxcn repo and `sudo pacman -S dtkcommon-git dtkcore-git dtkwidget-git dtkgui-git`, then reboot to resolve this issue temporary. We'll let you know when the issue gets fixed and you can then switch back to the official repo version once it's fixed.

HTH

**Reply by Cardellino** [reply]
Hello BLumia, 
the dock is visible and working again.
Thank you for your support.
Have a nice day..

**Reply by BLumia** [reply]
The fix should be land in Arch Linux official repo later today (you can check if your mirror have `dtkcore 1:5.6.2-1` update), after that you can safely switchback to the version from official repo. Make sure use `pacman -Qq | grep '\-git'` to check out if you have any other DTK related `-git` packages since they might be installed as dependency of the `-git` version of DTK packages you previously installed.

**Reply by Cardellino** [reply]
Thanks again Blumia.....
